netbench-orchestrator/src/orchestrator/cli.rs

```diff
@@ -0,0 +1,121 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::orchestrator::{OrchError, OrchResult};
+use serde::Deserialize;
+use std::{
+    fs::File,
+    path::{Path, PathBuf},
+};
+
+#[derive(Clone, Debug)]
+pub struct OrchestratorConfig {
+    // netbench
+    netbench_scenario_filename: String,
+    netbench_scenario_filepath: PathBuf,
+
+    // cdk
+    pub cdk_config: CdkConfig,
+}
+
+// Used for parsing the scenario file generated by the s2n-netbench project
+#[derive(Clone, Debug, Default, Deserialize)]
+pub struct CdkConfig {
```

Comment:
I don't entirely understand why we have this struct since it only has a single member? If it's just something for parsing, could we instead use serde_json to parse to a weakly typed json value and then get the appropriate object for the member key?

Reply:
It seems unfortunate to have our data modelling by entirely dictated by the `.json` structure. 

https://github.com/serde-rs/json?tab=readme-ov-file#operating-on-untyped-json-values

Could we use this to loosely deserialize the full object into a `JsonValue`, and use that to retrieve the inner `resources` object which can then be parsed into a strongly typed `CdkResources`?

Reply:
We could but I dont think its worth the effort or the complexity that comes with JsonValue. Its just simpler to match up the json file to the structs, especially since the json file might add new values(new infra) in the future.